srcpkgs/python-pyotp/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pyotp'
+pkgname=python-pyotp
+version=2.2.6
+revision=1
+noarch=yes
+wrksrc="pyotp-${version}"
+build_style=python-module
+pycompile_module="pyotp"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
+short_desc="Python2 One-Time Password Library"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="0BSD"
+homepage="https://github.com/pyotp/pyotp"
+distfiles="${PYPI_SITE}/p/pyotp/pyotp-${version}.tar.gz"
+checksum=dd9130dd91a0340d89a0f06f887dbd76dd07fb95a8886dc4bc401239f2eebd69
+
+do_check() {
+	python -B test.py
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-pyotp_package() {
+	short_desc="Python3 One-Time Password Library"
```

Comment:
you cna just short_desc="${short_desc/Python2/Python3}"

Reply:
Hm, I personally prefer keeping it simple by not doing stuff like that, I'll change it, seems like idiomatic Void